ansible/ansible-variables.yml

```diff
@@ -219,6 +219,10 @@ drivers_kubedriver_type: kubernetes
 drivers_kubedriver_url: https://kubedriver:8294
 drivers_kubedriver_cert_secret: kubedriver-tls
 
+# Enable to create a Deployment Location for kubedriver (in AIO Kubernetes cluster). This deployment location also supports connections to pod from Ansible Lifecycle Manager (ald version >= 2.0.1)
+deployment_location_kubedriver: True
+deployment_location_kubedriver_name: local-kubernetes
```

Comment:
Minor: might be useful to indicate that `deployment_location_kubedriver_name` is the `name` of the deployment location that is created (it wasn't obvious to me until I looked at the Ansible role source code).

Reply:
@sglover Do you have any suggestion for a different variable name?

Reply:
`kubedriver_deployment_location_name` is probably clearer. Perhaps also `kubedriver_create_deployment_location`?